### PR TITLE
osbuild: rpm metadata

### PR DESCRIPTION
**Pull request by teg**
Parse the metadata about the RPM stage as returned from osbuild.

### CODE REVIEW

File: internal/osbuild/result.go [hunk 1]
```diff
@@ -6,33 +6,74 @@ import (
...
 	Success bool            `json:"success"`
 	Output  string          `json:"output"`
 }
 
-type stage struct {
-	Name    string          `json:"name"`
-	Options json.RawMessage `json:"options"`
-	Success bool            `json:"success"`
-	Output  string          `json:"output"`
+type StageResult struct {
```
Comment (larskarlitski) [inline internal/osbuild/result.go:16]: Why did you make this public? It doesn't seem to be used anywhere outside this package.

**Review by larskarlitski — APPROVED** [review]
Great work, thanks! I have a small question, but don't block on that.

File: internal/osbuild/result.go (hunk 1 above)
Comment (teg) [reply internal/osbuild/result.go:16]: We need to translate this to the koji metadata format, not sure where the translation will live, but i assumed not in this package.

**Review by gicmo — APPROVED** [review]
:ok_person: Cool stuff!

**Review by ondrejbudai — APPROVED** [review]
Looks good, thanks @teg

**Pull request merged**